src/OpenTelemetry.Instrumentation.EntityFrameworkCore/CHANGELOG.md

```diff
@@ -8,6 +8,11 @@
 * Attribute `db.system` reports `oracle` when
   `Devart.Data.Oracle.Entity.EFCore` is used a provider.
   ([#2465](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2465))
+  
+* Added code to fix exception thrown when Devart Oracle is used
+(`this.commandTypeFetcher.Fetch(command)`). If Devart Oracle is used,
+the SQL statement will be extracted from the payload.
+([#2466](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2466))
```

Comment:
```suggestion
* Correctly report `db.statement`/`db.query.text` when
  Devart Oracle is used as a provider.
  ([#2466](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2466))
```

or something like that. Changelog is for end-users, It does not need to bring technical details how it was fixed. The PR link can brings additional details.